webapp/src/index.js

```diff
@@ -27,8 +27,16 @@ class Plugin {
             const iconURL = getPluginURL(store.getState()) + '/public/app-bar-icon.png';
             registry.registerAppBarComponent(
                 iconURL,
-                (channel) => {
-                    startMeeting(channel.id)(store.dispatch, store.getState);
+                async (channel) => {
+                    if (channel) {
+                        startMeeting(channel.id, '')(store.dispatch, store.getState);
+                    } else {
+                        const teamId = store.getState().entities.teams.currentTeamId;
+                        const threadId = store.getState().views.threads.selectedThreadIdInTeam[teamId];
+                        const baseURL = store.getState().entities.general.config.SiteURL;
```

Comment:
I would prefer these to be in a separate file for selectors, but it's fine to have them here like this. Just one suggestion to set the state to a local var

```suggestion
                        const state = store.getState();
                        const teamId = state.entities.teams.currentTeamId;
                        const threadId = state.views.threads.selectedThreadIdInTeam[teamId];
                        const baseURL = state.entities.general.config.SiteURL;
```